[2017-08-09 12:17:20,976][DEBUG][io.undertow.request.io] Error reading request
java.io.IOException: Connection reset by peer
	at sun.nio.ch.FileDispatcherImpl.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:39)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223)
	at sun.nio.ch.IOUtil.read(IOUtil.java:192)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:380)
	at org.xnio.nio.NioSocketConduit.read(NioSocketConduit.java:287)
	at org.xnio.conduits.ConduitStreamSourceChannel.read(ConduitStreamSourceChannel.java:127)
	at io.undertow.server.protocol.http.HttpReadListener.handleEventWithNoRunningRequest(HttpReadListener.java:150)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:128)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:56)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.conduits.ReadReadyHandler$ChannelListenerHandler.readReady(ReadReadyHandler.java:66)
	at org.xnio.nio.NioSocketConduit.handleReady(NioSocketConduit.java:89)
	at org.xnio.nio.WorkerThread.run(WorkerThread.java:545)
[2017-08-09 12:40:22,411][DEBUG][io.undertow.request.io] Error reading request
java.io.IOException: Connection reset by peer
	at sun.nio.ch.FileDispatcherImpl.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:39)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223)
	at sun.nio.ch.IOUtil.read(IOUtil.java:192)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:380)
	at org.xnio.nio.NioSocketConduit.read(NioSocketConduit.java:287)
	at org.xnio.conduits.ConduitStreamSourceChannel.read(ConduitStreamSourceChannel.java:127)
	at io.undertow.server.protocol.http.HttpReadListener.handleEventWithNoRunningRequest(HttpReadListener.java:150)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:128)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:56)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.conduits.ReadReadyHandler$ChannelListenerHandler.readReady(ReadReadyHandler.java:66)
	at org.xnio.nio.NioSocketConduit.handleReady(NioSocketConduit.java:89)
	at org.xnio.nio.WorkerThread.run(WorkerThread.java:545)
[2017-08-09 13:04:48,434][DEBUG][io.undertow.request.io] Error reading request
java.io.IOException: Connection reset by peer
	at sun.nio.ch.FileDispatcherImpl.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:39)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223)
	at sun.nio.ch.IOUtil.read(IOUtil.java:192)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:380)
	at org.xnio.nio.NioSocketConduit.read(NioSocketConduit.java:287)
	at org.xnio.conduits.ConduitStreamSourceChannel.read(ConduitStreamSourceChannel.java:127)
	at io.undertow.server.protocol.http.HttpReadListener.handleEventWithNoRunningRequest(HttpReadListener.java:150)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:128)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:56)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.conduits.ReadReadyHandler$ChannelListenerHandler.readReady(ReadReadyHandler.java:66)
	at org.xnio.nio.NioSocketConduit.handleReady(NioSocketConduit.java:89)
	at org.xnio.nio.WorkerThread.run(WorkerThread.java:545)
[2017-08-09 15:06:20,060][DEBUG][io.undertow.request.io] Error reading request
java.io.IOException: Connection reset by peer
	at sun.nio.ch.FileDispatcherImpl.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:39)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223)
	at sun.nio.ch.IOUtil.read(IOUtil.java:192)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:380)
	at org.xnio.nio.NioSocketConduit.read(NioSocketConduit.java:287)
	at org.xnio.conduits.ConduitStreamSourceChannel.read(ConduitStreamSourceChannel.java:127)
	at io.undertow.server.protocol.http.HttpReadListener.handleEventWithNoRunningRequest(HttpReadListener.java:150)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:128)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:56)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.conduits.ReadReadyHandler$ChannelListenerHandler.readReady(ReadReadyHandler.java:66)
	at org.xnio.nio.NioSocketConduit.handleReady(NioSocketConduit.java:89)
	at org.xnio.nio.WorkerThread.run(WorkerThread.java:545)
[2017-08-09 16:01:45,313][DEBUG][io.undertow.request.io] Error reading request
java.io.IOException: Connection reset by peer
	at sun.nio.ch.FileDispatcherImpl.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:39)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223)
	at sun.nio.ch.IOUtil.read(IOUtil.java:192)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:380)
	at org.xnio.nio.NioSocketConduit.read(NioSocketConduit.java:287)
	at org.xnio.conduits.ConduitStreamSourceChannel.read(ConduitStreamSourceChannel.java:127)
	at io.undertow.server.protocol.http.HttpReadListener.handleEventWithNoRunningRequest(HttpReadListener.java:150)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:128)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:56)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.conduits.ReadReadyHandler$ChannelListenerHandler.readReady(ReadReadyHandler.java:66)
	at org.xnio.nio.NioSocketConduit.handleReady(NioSocketConduit.java:89)
	at org.xnio.nio.WorkerThread.run(WorkerThread.java:545)
[2017-08-09 20:17:27,395][DEBUG][io.undertow.request.io] Error reading request
java.io.IOException: Connection reset by peer
	at sun.nio.ch.FileDispatcherImpl.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:39)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223)
	at sun.nio.ch.IOUtil.read(IOUtil.java:192)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:380)
	at org.xnio.nio.NioSocketConduit.read(NioSocketConduit.java:287)
	at org.xnio.conduits.ConduitStreamSourceChannel.read(ConduitStreamSourceChannel.java:127)
	at io.undertow.server.protocol.http.HttpReadListener.handleEventWithNoRunningRequest(HttpReadListener.java:150)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:128)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:56)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.conduits.ReadReadyHandler$ChannelListenerHandler.readReady(ReadReadyHandler.java:66)
	at org.xnio.nio.NioSocketConduit.handleReady(NioSocketConduit.java:89)
	at org.xnio.nio.WorkerThread.run(WorkerThread.java:545)
[2017-08-09 20:34:06,402][DEBUG][io.undertow.request.io] Error reading request
java.io.IOException: Connection reset by peer
	at sun.nio.ch.FileDispatcherImpl.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:39)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223)
	at sun.nio.ch.IOUtil.read(IOUtil.java:192)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:380)
	at org.xnio.nio.NioSocketConduit.read(NioSocketConduit.java:287)
	at org.xnio.conduits.ConduitStreamSourceChannel.read(ConduitStreamSourceChannel.java:127)
	at io.undertow.server.protocol.http.HttpReadListener.handleEventWithNoRunningRequest(HttpReadListener.java:150)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:128)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:56)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.conduits.ReadReadyHandler$ChannelListenerHandler.readReady(ReadReadyHandler.java:66)
	at org.xnio.nio.NioSocketConduit.handleReady(NioSocketConduit.java:89)
	at org.xnio.nio.WorkerThread.run(WorkerThread.java:545)
[2017-08-09 23:03:38,275][DEBUG][io.undertow.request.io] Error reading request
java.io.IOException: Connection reset by peer
	at sun.nio.ch.FileDispatcherImpl.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:39)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223)
	at sun.nio.ch.IOUtil.read(IOUtil.java:192)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:380)
	at org.xnio.nio.NioSocketConduit.read(NioSocketConduit.java:287)
	at org.xnio.conduits.ConduitStreamSourceChannel.read(ConduitStreamSourceChannel.java:127)
	at io.undertow.server.protocol.http.HttpReadListener.handleEventWithNoRunningRequest(HttpReadListener.java:150)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:128)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:56)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.conduits.ReadReadyHandler$ChannelListenerHandler.readReady(ReadReadyHandler.java:66)
	at org.xnio.nio.NioSocketConduit.handleReady(NioSocketConduit.java:89)
	at org.xnio.nio.WorkerThread.run(WorkerThread.java:545)
[2017-08-10 00:43:07,806][DEBUG][io.undertow.request.io] Error reading request
java.io.IOException: Connection reset by peer
	at sun.nio.ch.FileDispatcherImpl.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:39)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223)
	at sun.nio.ch.IOUtil.read(IOUtil.java:192)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:380)
	at org.xnio.nio.NioSocketConduit.read(NioSocketConduit.java:287)
	at org.xnio.conduits.ConduitStreamSourceChannel.read(ConduitStreamSourceChannel.java:127)
	at io.undertow.server.protocol.http.HttpReadListener.handleEventWithNoRunningRequest(HttpReadListener.java:150)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:128)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:56)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.conduits.ReadReadyHandler$ChannelListenerHandler.readReady(ReadReadyHandler.java:66)
	at org.xnio.nio.NioSocketConduit.handleReady(NioSocketConduit.java:89)
	at org.xnio.nio.WorkerThread.run(WorkerThread.java:545)
[2017-08-10 08:38:00,679][DEBUG][io.undertow.request.io] Error reading request
java.io.IOException: Connection reset by peer
	at sun.nio.ch.FileDispatcherImpl.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:39)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223)
	at sun.nio.ch.IOUtil.read(IOUtil.java:192)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:380)
	at org.xnio.nio.NioSocketConduit.read(NioSocketConduit.java:287)
	at org.xnio.conduits.ConduitStreamSourceChannel.read(ConduitStreamSourceChannel.java:127)
	at io.undertow.server.protocol.http.HttpReadListener.handleEventWithNoRunningRequest(HttpReadListener.java:150)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:128)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:56)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.conduits.ReadReadyHandler$ChannelListenerHandler.readReady(ReadReadyHandler.java:66)
	at org.xnio.nio.NioSocketConduit.handleReady(NioSocketConduit.java:89)
	at org.xnio.nio.WorkerThread.run(WorkerThread.java:545)
[2017-11-02 14:24:14,822][DEBUG][org.jboss.logging] Logging Provider: org.jboss.logging.Log4jLoggerProvider
[2017-11-02 14:24:15,952][INFO][luminus.repl-server] nREPL server started on port 7000
[2017-11-02 14:24:15,959][INFO][com.zaxxer.hikari.HikariDataSource] HikariPool-0 - is starting.
[2017-11-02 14:24:16,469][INFO][rna.handler] #'rna.db.core/*db* started
[2017-11-02 14:24:16,470][INFO][rna.config] 
-=[rna started successfully using the development profile]=-
[2017-11-02 14:24:16,561][INFO][org.xnio] XNIO version 3.4.0.Beta1
[2017-11-02 14:24:16,927][INFO][org.xnio.nio] XNIO NIO Implementation Version 3.4.0.Beta1
[2017-11-02 14:24:17,027][INFO][org.projectodd.wunderboss.web.Web] Registered web context /
[2017-11-02 14:24:17,028][INFO][luminus.http-server] server started on port 8080
[2017-11-02 14:38:30,296][DEBUG][org.jboss.logging] Logging Provider: org.jboss.logging.Log4jLoggerProvider
[2017-11-02 14:38:31,618][ERROR][luminus.repl-server] failed to start nREPL
java.net.BindException: Address already in use
	at java.net.PlainSocketImpl.socketBind(Native Method)
	at java.net.AbstractPlainSocketImpl.bind(AbstractPlainSocketImpl.java:387)
	at java.net.ServerSocket.bind(ServerSocket.java:375)
	at java.net.ServerSocket.bind(ServerSocket.java:329)
	at clojure.tools.nrepl.server$start_server.invokeStatic(server.clj:143)
	at clojure.tools.nrepl.server$start_server.doInvoke(server.clj:121)
	at clojure.lang.RestFn.invoke(RestFn.java:1096)
	at luminus.repl_server$start.invokeStatic(repl_server.clj:23)
	at luminus.repl_server$start.invoke(repl_server.clj:13)
	at rna.core$start_app.invokeStatic(core.clj:32)
	at rna.core$start_app.invoke(core.clj:26)
	at rna.core$_main.invokeStatic(core.clj:42)
	at rna.core$_main.doInvoke(core.clj:37)
	at clojure.lang.RestFn.invoke(RestFn.java:397)
	at clojure.lang.Var.invoke(Var.java:375)
	at user$eval13844.invokeStatic(form-init6424010838808256000.clj:1)
	at user$eval13844.invoke(form-init6424010838808256000.clj:1)
	at clojure.lang.Compiler.eval(Compiler.java:6927)
	at clojure.lang.Compiler.eval(Compiler.java:6917)
	at clojure.lang.Compiler.load(Compiler.java:7379)
	at clojure.lang.Compiler.loadFile(Compiler.java:7317)
	at clojure.main$load_script.invokeStatic(main.clj:275)
	at clojure.main$init_opt.invokeStatic(main.clj:277)
	at clojure.main$init_opt.invoke(main.clj:277)
	at clojure.main$initialize.invokeStatic(main.clj:308)
	at clojure.main$null_opt.invokeStatic(main.clj:342)
	at clojure.main$null_opt.invoke(main.clj:339)
	at clojure.main$main.invokeStatic(main.clj:421)
	at clojure.main$main.doInvoke(main.clj:384)
	at clojure.lang.RestFn.invoke(RestFn.java:421)
	at clojure.lang.Var.invoke(Var.java:383)
	at clojure.lang.AFn.applyToHelper(AFn.java:156)
	at clojure.lang.Var.applyTo(Var.java:700)
	at clojure.main.main(main.java:37)
[2017-11-02 14:38:31,625][INFO][com.zaxxer.hikari.HikariDataSource] HikariPool-0 - is starting.
[2017-11-02 14:38:31,954][INFO][rna.handler] #'rna.db.core/*db* started
[2017-11-02 14:38:31,954][INFO][rna.config] 
-=[rna started successfully using the development profile]=-
[2017-11-02 14:38:31,994][INFO][org.xnio] XNIO version 3.4.0.Beta1
[2017-11-02 14:38:32,062][INFO][org.xnio.nio] XNIO NIO Implementation Version 3.4.0.Beta1
[2017-11-02 14:38:32,164][ERROR][luminus.http-server] server failed to start on port 8080
java.lang.RuntimeException: java.net.BindException: Address already in use
	at io.undertow.Undertow.start(Undertow.java:183)
	at org.projectodd.wunderboss.web.undertow.UndertowWeb.start(UndertowWeb.java:83)
	at org.projectodd.wunderboss.web.undertow.UndertowWeb.registerHandler(UndertowWeb.java:143)
	at org.projectodd.wunderboss.web.undertow.UndertowWeb.registerHandler(UndertowWeb.java:124)
	at org.projectodd.wunderboss.web.undertow.UndertowWeb.registerHandler(UndertowWeb.java:68)
	at immutant.web.internal.wunderboss$mount.invokeStatic(wunderboss.clj:80)
	at immutant.web.internal.wunderboss$mount.invoke(wunderboss.clj:57)
	at immutant.web$run.invokeStatic(web.clj:97)
	at immutant.web$run.doInvoke(web.clj:23)
	at clojure.lang.RestFn.applyTo(RestFn.java:139)
	at clojure.core$apply.invokeStatic(core.clj:648)
	at clojure.core$apply.invoke(core.clj:641)
	at luminus.http_server$start.invokeStatic(http_server.clj:17)
	at luminus.http_server$start.invoke(http_server.clj:10)
	at rna.core$start_app.invokeStatic(core.clj:33)
	at rna.core$start_app.invoke(core.clj:26)
	at rna.core$_main.invokeStatic(core.clj:42)
	at rna.core$_main.doInvoke(core.clj:37)
	at clojure.lang.RestFn.invoke(RestFn.java:397)
	at clojure.lang.Var.invoke(Var.java:375)
	at user$eval13844.invokeStatic(form-init6424010838808256000.clj:1)
	at user$eval13844.invoke(form-init6424010838808256000.clj:1)
	at clojure.lang.Compiler.eval(Compiler.java:6927)
	at clojure.lang.Compiler.eval(Compiler.java:6917)
	at clojure.lang.Compiler.load(Compiler.java:7379)
	at clojure.lang.Compiler.loadFile(Compiler.java:7317)
	at clojure.main$load_script.invokeStatic(main.clj:275)
	at clojure.main$init_opt.invokeStatic(main.clj:277)
	at clojure.main$init_opt.invoke(main.clj:277)
	at clojure.main$initialize.invokeStatic(main.clj:308)
	at clojure.main$null_opt.invokeStatic(main.clj:342)
	at clojure.main$null_opt.invoke(main.clj:339)
	at clojure.main$main.invokeStatic(main.clj:421)
	at clojure.main$main.doInvoke(main.clj:384)
	at clojure.lang.RestFn.invoke(RestFn.java:421)
	at clojure.lang.Var.invoke(Var.java:383)
	at clojure.lang.AFn.applyToHelper(AFn.java:156)
	at clojure.lang.Var.applyTo(Var.java:700)
	at clojure.main.main(main.java:37)
Caused by: java.net.BindException: Address already in use
	at sun.nio.ch.Net.bind0(Native Method)
	at sun.nio.ch.Net.bind(Net.java:433)
	at sun.nio.ch.Net.bind(Net.java:425)
	at sun.nio.ch.ServerSocketChannelImpl.bind(ServerSocketChannelImpl.java:223)
	at sun.nio.ch.ServerSocketAdaptor.bind(ServerSocketAdaptor.java:74)
	at org.xnio.nio.NioXnioWorker.createTcpConnectionServer(NioXnioWorker.java:180)
	at org.xnio.XnioWorker.createStreamConnectionServer(XnioWorker.java:243)
	at io.undertow.Undertow.start(Undertow.java:139)
	... 38 more
[2017-11-02 16:35:34,407][INFO][luminus.repl-server] nREPL server stopped
[2017-11-02 16:35:34,409][INFO][rna.handler] rna is shutting down...
[2017-11-02 16:35:34,411][INFO][com.zaxxer.hikari.pool.HikariPool] HikariPool-0 - is closing down.
[2017-11-02 16:35:34,413][INFO][rna.handler] #'rna.db.core/*db* stopped
[2017-11-02 16:35:34,414][INFO][rna.handler] shutdown complete!
[2017-11-02 16:35:34,414][DEBUG][org.projectodd.wunderboss.WunderBoss] Returning existing component for org.projectodd.wunderboss.web.Web:115062141, ignoring options.
[2017-11-02 16:35:34,421][INFO][luminus.http-server] HTTP server stopped
[2017-11-13 16:37:49,204][WARN][com.zaxxer.hikari.pool.HikariPool] HikariPool-0 - Unusual system clock change detected, soft-evicting connections from pool.
[2018-01-17 10:31:50,854][DEBUG][org.jboss.logging] Logging Provider: org.jboss.logging.Log4jLoggerProvider
[2018-01-17 10:31:52,551][INFO][luminus.repl-server] nREPL server started on port 7000
[2018-01-17 10:31:52,582][INFO][com.zaxxer.hikari.HikariDataSource] HikariPool-0 - is starting.
[2018-01-17 10:31:53,092][INFO][rna.handler] #'rna.db.core/*db* started
[2018-01-17 10:31:53,093][INFO][rna.config] 
-=[rna started successfully using the development profile]=-
[2018-01-17 10:31:53,254][INFO][org.xnio] XNIO version 3.4.0.Beta1
[2018-01-17 10:31:53,773][INFO][org.xnio.nio] XNIO NIO Implementation Version 3.4.0.Beta1
[2018-01-17 10:31:53,957][INFO][org.projectodd.wunderboss.web.Web] Registered web context /
[2018-01-17 10:31:53,961][INFO][luminus.http-server] server started on port 8080
[2018-01-17 10:55:59,497][INFO][luminus.repl-server] nREPL server stopped
[2018-01-17 10:55:59,501][INFO][rna.handler] rna is shutting down...
[2018-01-17 10:55:59,505][INFO][com.zaxxer.hikari.pool.HikariPool] HikariPool-0 - is closing down.
[2018-01-17 10:55:59,514][INFO][rna.handler] #'rna.db.core/*db* stopped
[2018-01-17 10:55:59,515][INFO][rna.handler] shutdown complete!
[2018-01-17 10:55:59,516][DEBUG][org.projectodd.wunderboss.WunderBoss] Returning existing component for org.projectodd.wunderboss.web.Web:115062141, ignoring options.
[2018-01-17 10:55:59,528][INFO][luminus.http-server] HTTP server stopped
[2018-01-17 11:01:22,098][DEBUG][org.jboss.logging] Logging Provider: org.jboss.logging.Log4jLoggerProvider
[2018-01-17 11:01:23,422][INFO][luminus.repl-server] nREPL server started on port 7000
[2018-01-17 11:01:23,430][INFO][com.zaxxer.hikari.HikariDataSource] HikariPool-0 - is starting.
[2018-01-17 11:01:23,715][INFO][rna.handler] #'rna.db.core/*db* started
[2018-01-17 11:01:23,716][INFO][rna.config] 
-=[rna started successfully using the development profile]=-
[2018-01-17 11:01:23,766][INFO][org.xnio] XNIO version 3.4.0.Beta1
[2018-01-17 11:01:23,852][INFO][org.xnio.nio] XNIO NIO Implementation Version 3.4.0.Beta1
[2018-01-17 11:01:23,937][INFO][org.projectodd.wunderboss.web.Web] Registered web context /
[2018-01-17 11:01:23,938][INFO][luminus.http-server] server started on port 8080
[2018-01-18 04:48:06,072][DEBUG][io.undertow.request.io] Error reading request
java.io.IOException: Connection reset by peer
	at sun.nio.ch.FileDispatcherImpl.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:39)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223)
	at sun.nio.ch.IOUtil.read(IOUtil.java:192)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:380)
	at org.xnio.nio.NioSocketConduit.read(NioSocketConduit.java:287)
	at org.xnio.conduits.ConduitStreamSourceChannel.read(ConduitStreamSourceChannel.java:127)
	at io.undertow.server.protocol.http.HttpReadListener.handleEventWithNoRunningRequest(HttpReadListener.java:150)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:128)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:56)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.conduits.ReadReadyHandler$ChannelListenerHandler.readReady(ReadReadyHandler.java:66)
	at org.xnio.nio.NioSocketConduit.handleReady(NioSocketConduit.java:89)
	at org.xnio.nio.WorkerThread.run(WorkerThread.java:545)
[2018-01-18 07:56:41,196][DEBUG][io.undertow.request.io] Error reading request
java.io.IOException: Connection reset by peer
	at sun.nio.ch.FileDispatcherImpl.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:39)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223)
	at sun.nio.ch.IOUtil.read(IOUtil.java:192)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:380)
	at org.xnio.nio.NioSocketConduit.read(NioSocketConduit.java:287)
	at org.xnio.conduits.ConduitStreamSourceChannel.read(ConduitStreamSourceChannel.java:127)
	at io.undertow.server.protocol.http.HttpReadListener.handleEventWithNoRunningRequest(HttpReadListener.java:150)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:128)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:56)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.conduits.ReadReadyHandler$ChannelListenerHandler.readReady(ReadReadyHandler.java:66)
	at org.xnio.nio.NioSocketConduit.handleReady(NioSocketConduit.java:89)
	at org.xnio.nio.WorkerThread.run(WorkerThread.java:545)
[2018-01-19 12:11:02,469][DEBUG][io.undertow.request.io] Error reading request
java.io.IOException: Connection reset by peer
	at sun.nio.ch.FileDispatcherImpl.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:39)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223)
	at sun.nio.ch.IOUtil.read(IOUtil.java:192)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:380)
	at org.xnio.nio.NioSocketConduit.read(NioSocketConduit.java:287)
	at org.xnio.conduits.ConduitStreamSourceChannel.read(ConduitStreamSourceChannel.java:127)
	at io.undertow.server.protocol.http.HttpReadListener.handleEventWithNoRunningRequest(HttpReadListener.java:150)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:128)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:56)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.conduits.ReadReadyHandler$ChannelListenerHandler.readReady(ReadReadyHandler.java:66)
	at org.xnio.nio.NioSocketConduit.handleReady(NioSocketConduit.java:89)
	at org.xnio.nio.WorkerThread.run(WorkerThread.java:545)
[2018-01-19 15:51:23,071][DEBUG][io.undertow.request.io] Error reading request
java.io.IOException: Connection reset by peer
	at sun.nio.ch.FileDispatcherImpl.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:39)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223)
	at sun.nio.ch.IOUtil.read(IOUtil.java:192)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:380)
	at org.xnio.nio.NioSocketConduit.read(NioSocketConduit.java:287)
	at org.xnio.conduits.ConduitStreamSourceChannel.read(ConduitStreamSourceChannel.java:127)
	at io.undertow.server.protocol.http.HttpReadListener.handleEventWithNoRunningRequest(HttpReadListener.java:150)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:128)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:56)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.conduits.ReadReadyHandler$ChannelListenerHandler.readReady(ReadReadyHandler.java:66)
	at org.xnio.nio.NioSocketConduit.handleReady(NioSocketConduit.java:89)
	at org.xnio.nio.WorkerThread.run(WorkerThread.java:545)
[2018-01-19 16:33:22,939][DEBUG][io.undertow.request.io] Error reading request
java.io.IOException: Connection reset by peer
	at sun.nio.ch.FileDispatcherImpl.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:39)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223)
	at sun.nio.ch.IOUtil.read(IOUtil.java:192)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:380)
	at org.xnio.nio.NioSocketConduit.read(NioSocketConduit.java:287)
	at org.xnio.conduits.ConduitStreamSourceChannel.read(ConduitStreamSourceChannel.java:127)
	at io.undertow.server.protocol.http.HttpReadListener.handleEventWithNoRunningRequest(HttpReadListener.java:150)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:128)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:56)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.conduits.ReadReadyHandler$ChannelListenerHandler.readReady(ReadReadyHandler.java:66)
	at org.xnio.nio.NioSocketConduit.handleReady(NioSocketConduit.java:89)
	at org.xnio.nio.WorkerThread.run(WorkerThread.java:545)
[2018-01-20 00:16:35,258][DEBUG][io.undertow.request.io] Error reading request
java.io.IOException: Connection reset by peer
	at sun.nio.ch.FileDispatcherImpl.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:39)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223)
	at sun.nio.ch.IOUtil.read(IOUtil.java:192)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:380)
	at org.xnio.nio.NioSocketConduit.read(NioSocketConduit.java:287)
	at org.xnio.conduits.ConduitStreamSourceChannel.read(ConduitStreamSourceChannel.java:127)
	at io.undertow.server.protocol.http.HttpReadListener.handleEventWithNoRunningRequest(HttpReadListener.java:150)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:128)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:56)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.conduits.ReadReadyHandler$ChannelListenerHandler.readReady(ReadReadyHandler.java:66)
	at org.xnio.nio.NioSocketConduit.handleReady(NioSocketConduit.java:89)
	at org.xnio.nio.WorkerThread.run(WorkerThread.java:545)
[2018-01-20 01:48:56,213][DEBUG][io.undertow.request.io] Error reading request
java.io.IOException: Connection reset by peer
	at sun.nio.ch.FileDispatcherImpl.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:39)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223)
	at sun.nio.ch.IOUtil.read(IOUtil.java:192)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:380)
	at org.xnio.nio.NioSocketConduit.read(NioSocketConduit.java:287)
	at org.xnio.conduits.ConduitStreamSourceChannel.read(ConduitStreamSourceChannel.java:127)
	at io.undertow.server.protocol.http.HttpReadListener.handleEventWithNoRunningRequest(HttpReadListener.java:150)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:128)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:56)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.conduits.ReadReadyHandler$ChannelListenerHandler.readReady(ReadReadyHandler.java:66)
	at org.xnio.nio.NioSocketConduit.handleReady(NioSocketConduit.java:89)
	at org.xnio.nio.WorkerThread.run(WorkerThread.java:545)
[2018-01-20 06:43:08,076][DEBUG][io.undertow.request.io] Error reading request
java.io.IOException: Connection reset by peer
	at sun.nio.ch.FileDispatcherImpl.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:39)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223)
	at sun.nio.ch.IOUtil.read(IOUtil.java:192)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:380)
	at org.xnio.nio.NioSocketConduit.read(NioSocketConduit.java:287)
	at org.xnio.conduits.ConduitStreamSourceChannel.read(ConduitStreamSourceChannel.java:127)
	at io.undertow.server.protocol.http.HttpReadListener.handleEventWithNoRunningRequest(HttpReadListener.java:150)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:128)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:56)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.conduits.ReadReadyHandler$ChannelListenerHandler.readReady(ReadReadyHandler.java:66)
	at org.xnio.nio.NioSocketConduit.handleReady(NioSocketConduit.java:89)
	at org.xnio.nio.WorkerThread.run(WorkerThread.java:545)
[2018-01-20 12:31:10,660][DEBUG][io.undertow.request.io] UT005013: An IOException occurred
java.nio.channels.ClosedChannelException
	at sun.nio.ch.SocketChannelImpl.ensureWriteOpen(SocketChannelImpl.java:270)
	at sun.nio.ch.SocketChannelImpl.write(SocketChannelImpl.java:494)
	at org.xnio.nio.NioSocketConduit.write(NioSocketConduit.java:162)
	at io.undertow.server.protocol.http.HttpResponseConduit.write(HttpResponseConduit.java:621)
	at io.undertow.conduits.ChunkedStreamSinkConduit.doWrite(ChunkedStreamSinkConduit.java:169)
	at io.undertow.conduits.ChunkedStreamSinkConduit.writeFinal(ChunkedStreamSinkConduit.java:237)
	at org.xnio.conduits.ConduitStreamSinkChannel.writeFinal(ConduitStreamSinkChannel.java:104)
	at io.undertow.channels.DetachableStreamSinkChannel.writeFinal(DetachableStreamSinkChannel.java:195)
	at io.undertow.server.HttpServerExchange$WriteDispatchChannel.writeFinal(HttpServerExchange.java:1902)
	at io.undertow.io.UndertowOutputStream.writeBufferBlocking(UndertowOutputStream.java:291)
	at io.undertow.io.UndertowOutputStream.close(UndertowOutputStream.java:334)
	at io.undertow.server.HttpServerExchange$DefaultBlockingHttpExchange.close(HttpServerExchange.java:1768)
	at io.undertow.server.HttpServerExchange.endExchange(HttpServerExchange.java:1487)
	at io.undertow.server.AbstractServerConnection$CloseSetter.handleEvent(AbstractServerConnection.java:304)
	at io.undertow.server.AbstractServerConnection$CloseSetter.handleEvent(AbstractServerConnection.java:284)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.StreamConnection.invokeCloseListener(StreamConnection.java:80)
	at org.xnio.Connection.close(Connection.java:142)
	at io.undertow.server.AbstractServerConnection.close(AbstractServerConnection.java:147)
	at org.xnio.IoUtils.safeClose(IoUtils.java:152)
	at io.undertow.server.protocol.http.HttpResponseConduit.write(HttpResponseConduit.java:623)
	at io.undertow.conduits.ChunkedStreamSinkConduit.doWrite(ChunkedStreamSinkConduit.java:163)
	at io.undertow.conduits.ChunkedStreamSinkConduit.write(ChunkedStreamSinkConduit.java:127)
	at io.undertow.conduits.ChunkedStreamSinkConduit.write(ChunkedStreamSinkConduit.java:216)
	at org.xnio.conduits.ConduitStreamSinkChannel.write(ConduitStreamSinkChannel.java:158)
	at io.undertow.channels.DetachableStreamSinkChannel.write(DetachableStreamSinkChannel.java:179)
	at io.undertow.server.HttpServerExchange$WriteDispatchChannel.write(HttpServerExchange.java:1888)
	at org.xnio.channels.Channels.writeBlocking(Channels.java:152)
	at io.undertow.io.UndertowOutputStream.write(UndertowOutputStream.java:182)
	at io.undertow.io.UndertowOutputStream.write(UndertowOutputStream.java:102)
	at immutant.web.internal.ring$eval8568$fn__8569.invoke(ring.clj:126)
	at immutant.web.internal.ring$eval8539$fn__8540$G__8530__8549.invoke(ring.clj:112)
	at immutant.web.internal.ring$write_response.invokeStatic(ring.clj:153)
	at immutant.web.internal.ring$write_response.invoke(ring.clj:142)
	at immutant.web.internal.undertow$create_http_handler$reify__13570.handleRequest(undertow.clj:138)
	at org.projectodd.wunderboss.web.undertow.async.websocket.UndertowWebsocket$2.handleRequest(UndertowWebsocket.java:104)
	at io.undertow.server.session.SessionAttachmentHandler.handleRequest(SessionAttachmentHandler.java:68)
	at io.undertow.server.Connectors.executeRootHandler(Connectors.java:199)
	at io.undertow.server.HttpServerExchange$1.run(HttpServerExchange.java:778)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
[2018-01-20 12:31:10,661][ERROR][io.undertow.request] Undertow request failed HttpServerExchange{ POST /toptable request {Accept=[text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8], Accept-Language=[en-US,en;q=0.5], Accept-Encoding=[gzip, deflate], User-Agent=[Mozilla/5.0 (Macintosh; Intel Mac OS X 10.13; rv:57.0) Gecko/20100101 Firefox/57.0], Connection=[keep-alive], Content-Type=[application/x-www-form-urlencoded], Content-Length=[147], Cookie=[JSESSIONID=tIrG35EffQLNpkHOXKune6EdzwXRnINeKrRpfqlz], Referer=[http://tare.medisin.ntnu.no:8080/browse], Upgrade-Insecure-Requests=[1], Host=[tare.medisin.ntnu.no:8080]} response {Connection=[keep-alive], Server=[undertow], X-XSS-Protection=[1; mode=block], X-Content-Type-Options=[nosniff], X-Frame-Options=[SAMEORIGIN], Transfer-Encoding=[chunked], Content-Type=[text/html; charset=utf-8], Date=[Sat, 20 Jan 2018 11:31:03 GMT]}}
java.io.IOException: Broken pipe
	at sun.nio.ch.FileDispatcherImpl.writev0(Native Method)
	at sun.nio.ch.SocketDispatcher.writev(SocketDispatcher.java:51)
	at sun.nio.ch.IOUtil.write(IOUtil.java:148)
	at sun.nio.ch.SocketChannelImpl.write(SocketChannelImpl.java:504)
	at org.xnio.nio.NioSocketConduit.write(NioSocketConduit.java:162)
	at io.undertow.server.protocol.http.HttpResponseConduit.write(HttpResponseConduit.java:621)
	at io.undertow.conduits.ChunkedStreamSinkConduit.doWrite(ChunkedStreamSinkConduit.java:163)
	at io.undertow.conduits.ChunkedStreamSinkConduit.write(ChunkedStreamSinkConduit.java:127)
	at io.undertow.conduits.ChunkedStreamSinkConduit.write(ChunkedStreamSinkConduit.java:216)
	at org.xnio.conduits.ConduitStreamSinkChannel.write(ConduitStreamSinkChannel.java:158)
	at io.undertow.channels.DetachableStreamSinkChannel.write(DetachableStreamSinkChannel.java:179)
	at io.undertow.server.HttpServerExchange$WriteDispatchChannel.write(HttpServerExchange.java:1888)
	at org.xnio.channels.Channels.writeBlocking(Channels.java:152)
	at io.undertow.io.UndertowOutputStream.write(UndertowOutputStream.java:182)
	at io.undertow.io.UndertowOutputStream.write(UndertowOutputStream.java:102)
	at immutant.web.internal.ring$eval8568$fn__8569.invoke(ring.clj:126)
	at immutant.web.internal.ring$eval8539$fn__8540$G__8530__8549.invoke(ring.clj:112)
	at immutant.web.internal.ring$write_response.invokeStatic(ring.clj:153)
	at immutant.web.internal.ring$write_response.invoke(ring.clj:142)
	at immutant.web.internal.undertow$create_http_handler$reify__13570.handleRequest(undertow.clj:138)
	at org.projectodd.wunderboss.web.undertow.async.websocket.UndertowWebsocket$2.handleRequest(UndertowWebsocket.java:104)
	at io.undertow.server.session.SessionAttachmentHandler.handleRequest(SessionAttachmentHandler.java:68)
	at io.undertow.server.Connectors.executeRootHandler(Connectors.java:199)
	at io.undertow.server.HttpServerExchange$1.run(HttpServerExchange.java:778)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
[2018-01-20 15:50:53,454][DEBUG][io.undertow.request.io] Error reading request
java.io.IOException: Connection reset by peer
	at sun.nio.ch.FileDispatcherImpl.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:39)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223)
	at sun.nio.ch.IOUtil.read(IOUtil.java:192)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:380)
	at org.xnio.nio.NioSocketConduit.read(NioSocketConduit.java:287)
	at org.xnio.conduits.ConduitStreamSourceChannel.read(ConduitStreamSourceChannel.java:127)
	at io.undertow.server.protocol.http.HttpReadListener.handleEventWithNoRunningRequest(HttpReadListener.java:150)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:128)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:56)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.conduits.ReadReadyHandler$ChannelListenerHandler.readReady(ReadReadyHandler.java:66)
	at org.xnio.nio.NioSocketConduit.handleReady(NioSocketConduit.java:89)
	at org.xnio.nio.WorkerThread.run(WorkerThread.java:545)
[2018-01-20 17:47:55,715][DEBUG][io.undertow.request.io] Error reading request
java.io.IOException: Connection reset by peer
	at sun.nio.ch.FileDispatcherImpl.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:39)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223)
	at sun.nio.ch.IOUtil.read(IOUtil.java:192)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:380)
	at org.xnio.nio.NioSocketConduit.read(NioSocketConduit.java:287)
	at org.xnio.conduits.ConduitStreamSourceChannel.read(ConduitStreamSourceChannel.java:127)
	at io.undertow.server.protocol.http.HttpReadListener.handleEventWithNoRunningRequest(HttpReadListener.java:150)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:128)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:56)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.conduits.ReadReadyHandler$ChannelListenerHandler.readReady(ReadReadyHandler.java:66)
	at org.xnio.nio.NioSocketConduit.handleReady(NioSocketConduit.java:89)
	at org.xnio.nio.WorkerThread.run(WorkerThread.java:545)
[2018-01-20 21:05:06,250][DEBUG][io.undertow.request.io] Error reading request
java.io.IOException: Connection reset by peer
	at sun.nio.ch.FileDispatcherImpl.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:39)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223)
	at sun.nio.ch.IOUtil.read(IOUtil.java:192)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:380)
	at org.xnio.nio.NioSocketConduit.read(NioSocketConduit.java:287)
	at org.xnio.conduits.ConduitStreamSourceChannel.read(ConduitStreamSourceChannel.java:127)
	at io.undertow.server.protocol.http.HttpReadListener.handleEventWithNoRunningRequest(HttpReadListener.java:150)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:128)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:56)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.conduits.ReadReadyHandler$ChannelListenerHandler.readReady(ReadReadyHandler.java:66)
	at org.xnio.nio.NioSocketConduit.handleReady(NioSocketConduit.java:89)
	at org.xnio.nio.WorkerThread.run(WorkerThread.java:545)
[2018-01-20 21:50:30,526][DEBUG][io.undertow.request.io] UT005014: Failed to parse HTTP request
java.lang.IllegalArgumentException: Invalid string contents
	at io.undertow.util.HttpString.<init>(HttpString.java:111)
	at io.undertow.util.HttpString.<init>(HttpString.java:101)
	at io.undertow.server.protocol.http.HttpRequestParser$$generated.handleHttpVerb(Unknown Source)
	at io.undertow.server.protocol.http.HttpRequestParser.handle(HttpRequestParser.java:213)
	at io.undertow.server.protocol.http.HttpReadListener.handleEventWithNoRunningRequest(HttpReadListener.java:176)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:128)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:56)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.conduits.ReadReadyHandler$ChannelListenerHandler.readReady(ReadReadyHandler.java:66)
	at org.xnio.nio.NioSocketConduit.handleReady(NioSocketConduit.java:89)
	at org.xnio.nio.WorkerThread.run(WorkerThread.java:545)
[2018-01-20 21:50:34,170][DEBUG][io.undertow.request.io] Error reading request
java.io.IOException: Connection reset by peer
	at sun.nio.ch.FileDispatcherImpl.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:39)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223)
	at sun.nio.ch.IOUtil.read(IOUtil.java:192)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:380)
	at org.xnio.nio.NioSocketConduit.read(NioSocketConduit.java:287)
	at org.xnio.conduits.ConduitStreamSourceChannel.read(ConduitStreamSourceChannel.java:127)
	at io.undertow.server.protocol.http.HttpReadListener.handleEventWithNoRunningRequest(HttpReadListener.java:150)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:128)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:56)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.conduits.ReadReadyHandler$ChannelListenerHandler.readReady(ReadReadyHandler.java:66)
	at org.xnio.nio.NioSocketConduit.handleReady(NioSocketConduit.java:89)
	at org.xnio.nio.WorkerThread.run(WorkerThread.java:545)
[2018-01-20 21:50:34,772][DEBUG][io.undertow.request.io] Error reading request
java.io.IOException: Connection reset by peer
	at sun.nio.ch.FileDispatcherImpl.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:39)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223)
	at sun.nio.ch.IOUtil.read(IOUtil.java:192)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:380)
	at org.xnio.nio.NioSocketConduit.read(NioSocketConduit.java:287)
	at org.xnio.conduits.ConduitStreamSourceChannel.read(ConduitStreamSourceChannel.java:127)
	at io.undertow.server.protocol.http.HttpReadListener.handleEventWithNoRunningRequest(HttpReadListener.java:150)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:128)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:56)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.conduits.ReadReadyHandler$ChannelListenerHandler.readReady(ReadReadyHandler.java:66)
	at org.xnio.nio.NioSocketConduit.handleReady(NioSocketConduit.java:89)
	at org.xnio.nio.WorkerThread.run(WorkerThread.java:545)
[2018-01-21 06:53:09,855][DEBUG][io.undertow.request.io] Error reading request
java.io.IOException: Connection reset by peer
	at sun.nio.ch.FileDispatcherImpl.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:39)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223)
	at sun.nio.ch.IOUtil.read(IOUtil.java:192)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:380)
	at org.xnio.nio.NioSocketConduit.read(NioSocketConduit.java:287)
	at org.xnio.conduits.ConduitStreamSourceChannel.read(ConduitStreamSourceChannel.java:127)
	at io.undertow.server.protocol.http.HttpReadListener.handleEventWithNoRunningRequest(HttpReadListener.java:150)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:128)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:56)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.conduits.ReadReadyHandler$ChannelListenerHandler.readReady(ReadReadyHandler.java:66)
	at org.xnio.nio.NioSocketConduit.handleReady(NioSocketConduit.java:89)
	at org.xnio.nio.WorkerThread.run(WorkerThread.java:545)
[2018-01-21 07:30:04,145][DEBUG][io.undertow.request.io] Error reading request
java.io.IOException: Connection reset by peer
	at sun.nio.ch.FileDispatcherImpl.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:39)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223)
	at sun.nio.ch.IOUtil.read(IOUtil.java:192)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:380)
	at org.xnio.nio.NioSocketConduit.read(NioSocketConduit.java:287)
	at org.xnio.conduits.ConduitStreamSourceChannel.read(ConduitStreamSourceChannel.java:127)
	at io.undertow.server.protocol.http.HttpReadListener.handleEventWithNoRunningRequest(HttpReadListener.java:150)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:128)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:56)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.conduits.ReadReadyHandler$ChannelListenerHandler.readReady(ReadReadyHandler.java:66)
	at org.xnio.nio.NioSocketConduit.handleReady(NioSocketConduit.java:89)
	at org.xnio.nio.WorkerThread.run(WorkerThread.java:545)
[2018-01-21 09:18:33,133][DEBUG][io.undertow.request.io] Error reading request
java.io.IOException: Connection reset by peer
	at sun.nio.ch.FileDispatcherImpl.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:39)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223)
	at sun.nio.ch.IOUtil.read(IOUtil.java:192)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:380)
	at org.xnio.nio.NioSocketConduit.read(NioSocketConduit.java:287)
	at org.xnio.conduits.ConduitStreamSourceChannel.read(ConduitStreamSourceChannel.java:127)
	at io.undertow.server.protocol.http.HttpReadListener.handleEventWithNoRunningRequest(HttpReadListener.java:150)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:128)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:56)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.conduits.ReadReadyHandler$ChannelListenerHandler.readReady(ReadReadyHandler.java:66)
	at org.xnio.nio.NioSocketConduit.handleReady(NioSocketConduit.java:89)
	at org.xnio.nio.WorkerThread.run(WorkerThread.java:545)
[2018-01-21 10:17:26,462][DEBUG][io.undertow.request.io] Error reading request
java.io.IOException: Connection reset by peer
	at sun.nio.ch.FileDispatcherImpl.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:39)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223)
	at sun.nio.ch.IOUtil.read(IOUtil.java:192)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:380)
	at org.xnio.nio.NioSocketConduit.read(NioSocketConduit.java:287)
	at org.xnio.conduits.ConduitStreamSourceChannel.read(ConduitStreamSourceChannel.java:127)
	at io.undertow.server.protocol.http.HttpReadListener.handleEventWithNoRunningRequest(HttpReadListener.java:150)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:128)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:56)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.conduits.ReadReadyHandler$ChannelListenerHandler.readReady(ReadReadyHandler.java:66)
	at org.xnio.nio.NioSocketConduit.handleReady(NioSocketConduit.java:89)
	at org.xnio.nio.WorkerThread.run(WorkerThread.java:545)
[2018-01-21 11:06:38,232][DEBUG][io.undertow.request.io] Error reading request
java.io.IOException: Connection reset by peer
	at sun.nio.ch.FileDispatcherImpl.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:39)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223)
	at sun.nio.ch.IOUtil.read(IOUtil.java:192)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:380)
	at org.xnio.nio.NioSocketConduit.read(NioSocketConduit.java:287)
	at org.xnio.conduits.ConduitStreamSourceChannel.read(ConduitStreamSourceChannel.java:127)
	at io.undertow.server.protocol.http.HttpReadListener.handleEventWithNoRunningRequest(HttpReadListener.java:150)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:128)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:56)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.conduits.ReadReadyHandler$ChannelListenerHandler.readReady(ReadReadyHandler.java:66)
	at org.xnio.nio.NioSocketConduit.handleReady(NioSocketConduit.java:89)
	at org.xnio.nio.WorkerThread.run(WorkerThread.java:545)
[2018-01-21 11:27:06,186][DEBUG][io.undertow.request.io] Error reading request
java.io.IOException: Connection reset by peer
	at sun.nio.ch.FileDispatcherImpl.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:39)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223)
	at sun.nio.ch.IOUtil.read(IOUtil.java:192)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:380)
	at org.xnio.nio.NioSocketConduit.read(NioSocketConduit.java:287)
	at org.xnio.conduits.ConduitStreamSourceChannel.read(ConduitStreamSourceChannel.java:127)
	at io.undertow.server.protocol.http.HttpReadListener.handleEventWithNoRunningRequest(HttpReadListener.java:150)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:128)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:56)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.conduits.ReadReadyHandler$ChannelListenerHandler.readReady(ReadReadyHandler.java:66)
	at org.xnio.nio.NioSocketConduit.handleReady(NioSocketConduit.java:89)
	at org.xnio.nio.WorkerThread.run(WorkerThread.java:545)
[2018-01-22 07:09:12,549][DEBUG][io.undertow.request.io] Error reading request
java.io.IOException: Connection reset by peer
	at sun.nio.ch.FileDispatcherImpl.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:39)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223)
	at sun.nio.ch.IOUtil.read(IOUtil.java:192)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:380)
	at org.xnio.nio.NioSocketConduit.read(NioSocketConduit.java:287)
	at org.xnio.conduits.ConduitStreamSourceChannel.read(ConduitStreamSourceChannel.java:127)
	at io.undertow.server.protocol.http.HttpReadListener.handleEventWithNoRunningRequest(HttpReadListener.java:150)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:128)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:56)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.conduits.ReadReadyHandler$ChannelListenerHandler.readReady(ReadReadyHandler.java:66)
	at org.xnio.nio.NioSocketConduit.handleReady(NioSocketConduit.java:89)
	at org.xnio.nio.WorkerThread.run(WorkerThread.java:545)
[2018-01-22 07:27:26,407][DEBUG][io.undertow.request.io] Error reading request
java.io.IOException: Connection reset by peer
	at sun.nio.ch.FileDispatcherImpl.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:39)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223)
	at sun.nio.ch.IOUtil.read(IOUtil.java:192)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:380)
	at org.xnio.nio.NioSocketConduit.read(NioSocketConduit.java:287)
	at org.xnio.conduits.ConduitStreamSourceChannel.read(ConduitStreamSourceChannel.java:127)
	at io.undertow.server.protocol.http.HttpReadListener.handleEventWithNoRunningRequest(HttpReadListener.java:150)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:128)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:56)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.conduits.ReadReadyHandler$ChannelListenerHandler.readReady(ReadReadyHandler.java:66)
	at org.xnio.nio.NioSocketConduit.handleReady(NioSocketConduit.java:89)
	at org.xnio.nio.WorkerThread.run(WorkerThread.java:545)
[2018-01-22 09:50:14,049][DEBUG][io.undertow.request.io] Error reading request
java.io.IOException: Connection reset by peer
	at sun.nio.ch.FileDispatcherImpl.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:39)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223)
	at sun.nio.ch.IOUtil.read(IOUtil.java:192)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:380)
	at org.xnio.nio.NioSocketConduit.read(NioSocketConduit.java:287)
	at org.xnio.conduits.ConduitStreamSourceChannel.read(ConduitStreamSourceChannel.java:127)
	at io.undertow.server.protocol.http.HttpReadListener.handleEventWithNoRunningRequest(HttpReadListener.java:150)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:128)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:56)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.conduits.ReadReadyHandler$ChannelListenerHandler.readReady(ReadReadyHandler.java:66)
	at org.xnio.nio.NioSocketConduit.handleReady(NioSocketConduit.java:89)
	at org.xnio.nio.WorkerThread.run(WorkerThread.java:545)
[2018-01-22 09:50:15,876][DEBUG][io.undertow.request.io] Error reading request
java.io.IOException: Connection reset by peer
	at sun.nio.ch.FileDispatcherImpl.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:39)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223)
	at sun.nio.ch.IOUtil.read(IOUtil.java:192)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:380)
	at org.xnio.nio.NioSocketConduit.read(NioSocketConduit.java:287)
	at org.xnio.conduits.ConduitStreamSourceChannel.read(ConduitStreamSourceChannel.java:127)
	at io.undertow.server.protocol.http.HttpReadListener.handleEventWithNoRunningRequest(HttpReadListener.java:150)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:128)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:56)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.conduits.ReadReadyHandler$ChannelListenerHandler.readReady(ReadReadyHandler.java:66)
	at org.xnio.nio.NioSocketConduit.handleReady(NioSocketConduit.java:89)
	at org.xnio.nio.WorkerThread.run(WorkerThread.java:545)
[2018-01-22 19:55:58,808][DEBUG][io.undertow.request.io] Error reading request
java.io.IOException: Connection reset by peer
	at sun.nio.ch.FileDispatcherImpl.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:39)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223)
	at sun.nio.ch.IOUtil.read(IOUtil.java:192)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:380)
	at org.xnio.nio.NioSocketConduit.read(NioSocketConduit.java:287)
	at org.xnio.conduits.ConduitStreamSourceChannel.read(ConduitStreamSourceChannel.java:127)
	at io.undertow.server.protocol.http.HttpReadListener.handleEventWithNoRunningRequest(HttpReadListener.java:150)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:128)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:56)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.conduits.ReadReadyHandler$ChannelListenerHandler.readReady(ReadReadyHandler.java:66)
	at org.xnio.nio.NioSocketConduit.handleReady(NioSocketConduit.java:89)
	at org.xnio.nio.WorkerThread.run(WorkerThread.java:545)
[2018-01-22 19:56:13,845][DEBUG][io.undertow.request.io] Error reading request
java.io.IOException: Connection reset by peer
	at sun.nio.ch.FileDispatcherImpl.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:39)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223)
	at sun.nio.ch.IOUtil.read(IOUtil.java:192)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:380)
	at org.xnio.nio.NioSocketConduit.read(NioSocketConduit.java:287)
	at org.xnio.conduits.ConduitStreamSourceChannel.read(ConduitStreamSourceChannel.java:127)
	at io.undertow.server.protocol.http.HttpReadListener.handleEventWithNoRunningRequest(HttpReadListener.java:150)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:128)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:56)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.conduits.ReadReadyHandler$ChannelListenerHandler.readReady(ReadReadyHandler.java:66)
	at org.xnio.nio.NioSocketConduit.handleReady(NioSocketConduit.java:89)
	at org.xnio.nio.WorkerThread.run(WorkerThread.java:545)
[2018-01-22 21:25:01,530][DEBUG][io.undertow.request.io] UT005014: Failed to parse HTTP request
java.lang.IllegalArgumentException: Invalid string contents
	at io.undertow.util.HttpString.<init>(HttpString.java:111)
	at io.undertow.util.HttpString.<init>(HttpString.java:101)
	at io.undertow.server.protocol.http.HttpRequestParser$$generated.handleHttpVerb(Unknown Source)
	at io.undertow.server.protocol.http.HttpRequestParser.handle(HttpRequestParser.java:213)
	at io.undertow.server.protocol.http.HttpReadListener.handleEventWithNoRunningRequest(HttpReadListener.java:176)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:128)
	at io.undertow.server.protocol.http.HttpOpenListener.handleEvent(HttpOpenListener.java:143)
	at io.undertow.server.protocol.http.HttpOpenListener.handleEvent(HttpOpenListener.java:90)
	at io.undertow.server.protocol.http.HttpOpenListener.handleEvent(HttpOpenListener.java:49)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.ChannelListeners$10.handleEvent(ChannelListeners.java:291)
	at org.xnio.ChannelListeners$10.handleEvent(ChannelListeners.java:286)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.nio.NioTcpServerHandle.handleReady(NioTcpServerHandle.java:53)
	at org.xnio.nio.WorkerThread.run(WorkerThread.java:545)
[2018-01-22 22:19:45,485][DEBUG][io.undertow.request.io] Error reading request
java.io.IOException: Connection reset by peer
	at sun.nio.ch.FileDispatcherImpl.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:39)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223)
	at sun.nio.ch.IOUtil.read(IOUtil.java:192)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:380)
	at org.xnio.nio.NioSocketConduit.read(NioSocketConduit.java:287)
	at org.xnio.conduits.ConduitStreamSourceChannel.read(ConduitStreamSourceChannel.java:127)
	at io.undertow.server.protocol.http.HttpReadListener.handleEventWithNoRunningRequest(HttpReadListener.java:150)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:128)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:56)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.conduits.ReadReadyHandler$ChannelListenerHandler.readReady(ReadReadyHandler.java:66)
	at org.xnio.nio.NioSocketConduit.handleReady(NioSocketConduit.java:89)
	at org.xnio.nio.WorkerThread.run(WorkerThread.java:545)
[2018-01-23 14:16:28,180][DEBUG][io.undertow.request.io] Error reading request
java.io.IOException: Connection reset by peer
	at sun.nio.ch.FileDispatcherImpl.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:39)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223)
	at sun.nio.ch.IOUtil.read(IOUtil.java:192)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:380)
	at org.xnio.nio.NioSocketConduit.read(NioSocketConduit.java:287)
	at org.xnio.conduits.ConduitStreamSourceChannel.read(ConduitStreamSourceChannel.java:127)
	at io.undertow.server.protocol.http.HttpReadListener.handleEventWithNoRunningRequest(HttpReadListener.java:150)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:128)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:56)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.conduits.ReadReadyHandler$ChannelListenerHandler.readReady(ReadReadyHandler.java:66)
	at org.xnio.nio.NioSocketConduit.handleReady(NioSocketConduit.java:89)
	at org.xnio.nio.WorkerThread.run(WorkerThread.java:545)
[2018-01-24 02:21:51,068][DEBUG][io.undertow.request.io] UT005014: Failed to parse HTTP request
java.lang.IllegalArgumentException: Invalid string contents
	at io.undertow.util.HttpString.<init>(HttpString.java:111)
	at io.undertow.util.HttpString.<init>(HttpString.java:101)
	at io.undertow.server.protocol.http.HttpRequestParser$$generated.handleHttpVerb(Unknown Source)
	at io.undertow.server.protocol.http.HttpRequestParser.handle(HttpRequestParser.java:213)
	at io.undertow.server.protocol.http.HttpReadListener.handleEventWithNoRunningRequest(HttpReadListener.java:176)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:128)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:56)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.conduits.ReadReadyHandler$ChannelListenerHandler.readReady(ReadReadyHandler.java:66)
	at org.xnio.nio.NioSocketConduit.handleReady(NioSocketConduit.java:89)
	at org.xnio.nio.WorkerThread.run(WorkerThread.java:545)
[2018-01-24 02:36:34,105][DEBUG][io.undertow.request.io] Error reading request
java.io.IOException: Connection reset by peer
	at sun.nio.ch.FileDispatcherImpl.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:39)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223)
	at sun.nio.ch.IOUtil.read(IOUtil.java:192)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:380)
	at org.xnio.nio.NioSocketConduit.read(NioSocketConduit.java:287)
	at org.xnio.conduits.ConduitStreamSourceChannel.read(ConduitStreamSourceChannel.java:127)
	at io.undertow.server.protocol.http.HttpReadListener.handleEventWithNoRunningRequest(HttpReadListener.java:150)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:128)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:56)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.conduits.ReadReadyHandler$ChannelListenerHandler.readReady(ReadReadyHandler.java:66)
	at org.xnio.nio.NioSocketConduit.handleReady(NioSocketConduit.java:89)
	at org.xnio.nio.WorkerThread.run(WorkerThread.java:545)
[2018-01-24 05:45:56,094][DEBUG][io.undertow.request.io] Error reading request
java.io.IOException: Connection reset by peer
	at sun.nio.ch.FileDispatcherImpl.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:39)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223)
	at sun.nio.ch.IOUtil.read(IOUtil.java:192)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:380)
	at org.xnio.nio.NioSocketConduit.read(NioSocketConduit.java:287)
	at org.xnio.conduits.ConduitStreamSourceChannel.read(ConduitStreamSourceChannel.java:127)
	at io.undertow.server.protocol.http.HttpReadListener.handleEventWithNoRunningRequest(HttpReadListener.java:150)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:128)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:56)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.conduits.ReadReadyHandler$ChannelListenerHandler.readReady(ReadReadyHandler.java:66)
	at org.xnio.nio.NioSocketConduit.handleReady(NioSocketConduit.java:89)
	at org.xnio.nio.WorkerThread.run(WorkerThread.java:545)
[2018-01-24 05:53:28,247][DEBUG][io.undertow.request.io] Error reading request
java.io.IOException: Connection reset by peer
	at sun.nio.ch.FileDispatcherImpl.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:39)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223)
	at sun.nio.ch.IOUtil.read(IOUtil.java:192)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:380)
	at org.xnio.nio.NioSocketConduit.read(NioSocketConduit.java:287)
	at org.xnio.conduits.ConduitStreamSourceChannel.read(ConduitStreamSourceChannel.java:127)
	at io.undertow.server.protocol.http.HttpReadListener.handleEventWithNoRunningRequest(HttpReadListener.java:150)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:128)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:56)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.conduits.ReadReadyHandler$ChannelListenerHandler.readReady(ReadReadyHandler.java:66)
	at org.xnio.nio.NioSocketConduit.handleReady(NioSocketConduit.java:89)
	at org.xnio.nio.WorkerThread.run(WorkerThread.java:545)
[2018-01-24 06:18:09,884][DEBUG][io.undertow.request.io] Error reading request
java.io.IOException: Connection reset by peer
	at sun.nio.ch.FileDispatcherImpl.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:39)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223)
	at sun.nio.ch.IOUtil.read(IOUtil.java:192)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:380)
	at org.xnio.nio.NioSocketConduit.read(NioSocketConduit.java:287)
	at org.xnio.conduits.ConduitStreamSourceChannel.read(ConduitStreamSourceChannel.java:127)
	at io.undertow.server.protocol.http.HttpReadListener.handleEventWithNoRunningRequest(HttpReadListener.java:150)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:128)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:56)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.conduits.ReadReadyHandler$ChannelListenerHandler.readReady(ReadReadyHandler.java:66)
	at org.xnio.nio.NioSocketConduit.handleReady(NioSocketConduit.java:89)
	at org.xnio.nio.WorkerThread.run(WorkerThread.java:545)
[2018-01-24 10:52:33,765][DEBUG][io.undertow.request.io] Error reading request
java.io.IOException: Connection reset by peer
	at sun.nio.ch.FileDispatcherImpl.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:39)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223)
	at sun.nio.ch.IOUtil.read(IOUtil.java:192)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:380)
	at org.xnio.nio.NioSocketConduit.read(NioSocketConduit.java:287)
	at org.xnio.conduits.ConduitStreamSourceChannel.read(ConduitStreamSourceChannel.java:127)
	at io.undertow.server.protocol.http.HttpReadListener.handleEventWithNoRunningRequest(HttpReadListener.java:150)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:128)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:56)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.conduits.ReadReadyHandler$ChannelListenerHandler.readReady(ReadReadyHandler.java:66)
	at org.xnio.nio.NioSocketConduit.handleReady(NioSocketConduit.java:89)
	at org.xnio.nio.WorkerThread.run(WorkerThread.java:545)
[2018-01-24 13:00:48,321][DEBUG][io.undertow.request.io] UT005014: Failed to parse HTTP request
java.lang.IllegalArgumentException: Invalid string contents
	at io.undertow.util.HttpString.<init>(HttpString.java:111)
	at io.undertow.util.HttpString.<init>(HttpString.java:101)
	at io.undertow.server.protocol.http.HttpRequestParser$$generated.handleHttpVerb(Unknown Source)
	at io.undertow.server.protocol.http.HttpRequestParser.handle(HttpRequestParser.java:213)
	at io.undertow.server.protocol.http.HttpReadListener.handleEventWithNoRunningRequest(HttpReadListener.java:176)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:128)
	at io.undertow.server.protocol.http.HttpOpenListener.handleEvent(HttpOpenListener.java:143)
	at io.undertow.server.protocol.http.HttpOpenListener.handleEvent(HttpOpenListener.java:90)
	at io.undertow.server.protocol.http.HttpOpenListener.handleEvent(HttpOpenListener.java:49)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.ChannelListeners$10.handleEvent(ChannelListeners.java:291)
	at org.xnio.ChannelListeners$10.handleEvent(ChannelListeners.java:286)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.nio.NioTcpServerHandle.handleReady(NioTcpServerHandle.java:53)
	at org.xnio.nio.WorkerThread.run(WorkerThread.java:545)
[2018-01-24 13:11:50,563][DEBUG][io.undertow.request.io] Error reading request
java.io.IOException: Connection reset by peer
	at sun.nio.ch.FileDispatcherImpl.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:39)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223)
	at sun.nio.ch.IOUtil.read(IOUtil.java:192)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:380)
	at org.xnio.nio.NioSocketConduit.read(NioSocketConduit.java:287)
	at org.xnio.conduits.ConduitStreamSourceChannel.read(ConduitStreamSourceChannel.java:127)
	at io.undertow.server.protocol.http.HttpReadListener.handleEventWithNoRunningRequest(HttpReadListener.java:150)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:128)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:56)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.conduits.ReadReadyHandler$ChannelListenerHandler.readReady(ReadReadyHandler.java:66)
	at org.xnio.nio.NioSocketConduit.handleReady(NioSocketConduit.java:89)
	at org.xnio.nio.WorkerThread.run(WorkerThread.java:545)
[2018-01-24 18:00:16,091][DEBUG][io.undertow.request.io] Error reading request
java.io.IOException: Connection reset by peer
	at sun.nio.ch.FileDispatcherImpl.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:39)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223)
	at sun.nio.ch.IOUtil.read(IOUtil.java:192)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:380)
	at org.xnio.nio.NioSocketConduit.read(NioSocketConduit.java:287)
	at org.xnio.conduits.ConduitStreamSourceChannel.read(ConduitStreamSourceChannel.java:127)
	at io.undertow.server.protocol.http.HttpReadListener.handleEventWithNoRunningRequest(HttpReadListener.java:150)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:128)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:56)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.conduits.ReadReadyHandler$ChannelListenerHandler.readReady(ReadReadyHandler.java:66)
	at org.xnio.nio.NioSocketConduit.handleReady(NioSocketConduit.java:89)
	at org.xnio.nio.WorkerThread.run(WorkerThread.java:545)
[2018-01-24 19:37:48,717][DEBUG][io.undertow.request.io] Error reading request
java.io.IOException: Connection reset by peer
	at sun.nio.ch.FileDispatcherImpl.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:39)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223)
	at sun.nio.ch.IOUtil.read(IOUtil.java:192)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:380)
	at org.xnio.nio.NioSocketConduit.read(NioSocketConduit.java:287)
	at org.xnio.conduits.ConduitStreamSourceChannel.read(ConduitStreamSourceChannel.java:127)
	at io.undertow.server.protocol.http.HttpReadListener.handleEventWithNoRunningRequest(HttpReadListener.java:150)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:128)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:56)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.conduits.ReadReadyHandler$ChannelListenerHandler.readReady(ReadReadyHandler.java:66)
	at org.xnio.nio.NioSocketConduit.handleReady(NioSocketConduit.java:89)
	at org.xnio.nio.WorkerThread.run(WorkerThread.java:545)
[2018-01-24 21:11:46,203][DEBUG][io.undertow.request.io] Error reading request
java.io.IOException: Connection reset by peer
	at sun.nio.ch.FileDispatcherImpl.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:39)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223)
	at sun.nio.ch.IOUtil.read(IOUtil.java:192)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:380)
	at org.xnio.nio.NioSocketConduit.read(NioSocketConduit.java:287)
	at org.xnio.conduits.ConduitStreamSourceChannel.read(ConduitStreamSourceChannel.java:127)
	at io.undertow.server.protocol.http.HttpReadListener.handleEventWithNoRunningRequest(HttpReadListener.java:150)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:128)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:56)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.conduits.ReadReadyHandler$ChannelListenerHandler.readReady(ReadReadyHandler.java:66)
	at org.xnio.nio.NioSocketConduit.handleReady(NioSocketConduit.java:89)
	at org.xnio.nio.WorkerThread.run(WorkerThread.java:545)
[2018-01-24 21:16:13,461][DEBUG][io.undertow.request.io] Error reading request
java.io.IOException: Connection reset by peer
	at sun.nio.ch.FileDispatcherImpl.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:39)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223)
	at sun.nio.ch.IOUtil.read(IOUtil.java:192)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:380)
	at org.xnio.nio.NioSocketConduit.read(NioSocketConduit.java:287)
	at org.xnio.conduits.ConduitStreamSourceChannel.read(ConduitStreamSourceChannel.java:127)
	at io.undertow.server.protocol.http.HttpReadListener.handleEventWithNoRunningRequest(HttpReadListener.java:150)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:128)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:56)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.conduits.ReadReadyHandler$ChannelListenerHandler.readReady(ReadReadyHandler.java:66)
	at org.xnio.nio.NioSocketConduit.handleReady(NioSocketConduit.java:89)
	at org.xnio.nio.WorkerThread.run(WorkerThread.java:545)
[2018-01-24 22:44:22,225][DEBUG][io.undertow.request.io] Error reading request
java.io.IOException: Connection reset by peer
	at sun.nio.ch.FileDispatcherImpl.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:39)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223)
	at sun.nio.ch.IOUtil.read(IOUtil.java:192)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:380)
	at org.xnio.nio.NioSocketConduit.read(NioSocketConduit.java:287)
	at org.xnio.conduits.ConduitStreamSourceChannel.read(ConduitStreamSourceChannel.java:127)
	at io.undertow.server.protocol.http.HttpReadListener.handleEventWithNoRunningRequest(HttpReadListener.java:150)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:128)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:56)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.conduits.ReadReadyHandler$ChannelListenerHandler.readReady(ReadReadyHandler.java:66)
	at org.xnio.nio.NioSocketConduit.handleReady(NioSocketConduit.java:89)
	at org.xnio.nio.WorkerThread.run(WorkerThread.java:545)
[2018-01-25 08:14:13,223][DEBUG][io.undertow.request.io] Error reading request
java.io.IOException: Connection reset by peer
	at sun.nio.ch.FileDispatcherImpl.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:39)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223)
	at sun.nio.ch.IOUtil.read(IOUtil.java:192)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:380)
	at org.xnio.nio.NioSocketConduit.read(NioSocketConduit.java:287)
	at org.xnio.conduits.ConduitStreamSourceChannel.read(ConduitStreamSourceChannel.java:127)
	at io.undertow.server.protocol.http.HttpReadListener.handleEventWithNoRunningRequest(HttpReadListener.java:150)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:128)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:56)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.conduits.ReadReadyHandler$ChannelListenerHandler.readReady(ReadReadyHandler.java:66)
	at org.xnio.nio.NioSocketConduit.handleReady(NioSocketConduit.java:89)
	at org.xnio.nio.WorkerThread.run(WorkerThread.java:545)
[2018-01-25 08:47:36,415][DEBUG][io.undertow.request.io] Error reading request
java.io.IOException: Connection reset by peer
	at sun.nio.ch.FileDispatcherImpl.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:39)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223)
	at sun.nio.ch.IOUtil.read(IOUtil.java:192)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:380)
	at org.xnio.nio.NioSocketConduit.read(NioSocketConduit.java:287)
	at org.xnio.conduits.ConduitStreamSourceChannel.read(ConduitStreamSourceChannel.java:127)
	at io.undertow.server.protocol.http.HttpReadListener.handleEventWithNoRunningRequest(HttpReadListener.java:150)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:128)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:56)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.conduits.ReadReadyHandler$ChannelListenerHandler.readReady(ReadReadyHandler.java:66)
	at org.xnio.nio.NioSocketConduit.handleReady(NioSocketConduit.java:89)
	at org.xnio.nio.WorkerThread.run(WorkerThread.java:545)
[2018-01-25 09:58:13,665][DEBUG][io.undertow.request.io] Error reading request
java.io.IOException: Connection reset by peer
	at sun.nio.ch.FileDispatcherImpl.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:39)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223)
	at sun.nio.ch.IOUtil.read(IOUtil.java:192)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:380)
	at org.xnio.nio.NioSocketConduit.read(NioSocketConduit.java:287)
	at org.xnio.conduits.ConduitStreamSourceChannel.read(ConduitStreamSourceChannel.java:127)
	at io.undertow.server.protocol.http.HttpReadListener.handleEventWithNoRunningRequest(HttpReadListener.java:150)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:128)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:56)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.conduits.ReadReadyHandler$ChannelListenerHandler.readReady(ReadReadyHandler.java:66)
	at org.xnio.nio.NioSocketConduit.handleReady(NioSocketConduit.java:89)
	at org.xnio.nio.WorkerThread.run(WorkerThread.java:545)
[2018-01-25 12:27:10,202][DEBUG][io.undertow.request.io] Error reading request
java.io.IOException: Connection reset by peer
	at sun.nio.ch.FileDispatcherImpl.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:39)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223)
	at sun.nio.ch.IOUtil.read(IOUtil.java:192)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:380)
	at org.xnio.nio.NioSocketConduit.read(NioSocketConduit.java:287)
	at org.xnio.conduits.ConduitStreamSourceChannel.read(ConduitStreamSourceChannel.java:127)
	at io.undertow.server.protocol.http.HttpReadListener.handleEventWithNoRunningRequest(HttpReadListener.java:150)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:128)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:56)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.conduits.ReadReadyHandler$ChannelListenerHandler.readReady(ReadReadyHandler.java:66)
	at org.xnio.nio.NioSocketConduit.handleReady(NioSocketConduit.java:89)
	at org.xnio.nio.WorkerThread.run(WorkerThread.java:545)
[2018-01-25 19:35:16,864][DEBUG][io.undertow.request.io] Error reading request
java.io.IOException: Connection reset by peer
	at sun.nio.ch.FileDispatcherImpl.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:39)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223)
	at sun.nio.ch.IOUtil.read(IOUtil.java:192)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:380)
	at org.xnio.nio.NioSocketConduit.read(NioSocketConduit.java:287)
	at org.xnio.conduits.ConduitStreamSourceChannel.read(ConduitStreamSourceChannel.java:127)
	at io.undertow.server.protocol.http.HttpReadListener.handleEventWithNoRunningRequest(HttpReadListener.java:150)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:128)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:56)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.conduits.ReadReadyHandler$ChannelListenerHandler.readReady(ReadReadyHandler.java:66)
	at org.xnio.nio.NioSocketConduit.handleReady(NioSocketConduit.java:89)
	at org.xnio.nio.WorkerThread.run(WorkerThread.java:545)
[2018-01-25 21:25:34,183][DEBUG][io.undertow.request.io] Error reading request
java.io.IOException: Connection reset by peer
	at sun.nio.ch.FileDispatcherImpl.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:39)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223)
	at sun.nio.ch.IOUtil.read(IOUtil.java:192)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:380)
	at org.xnio.nio.NioSocketConduit.read(NioSocketConduit.java:287)
	at org.xnio.conduits.ConduitStreamSourceChannel.read(ConduitStreamSourceChannel.java:127)
	at io.undertow.server.protocol.http.HttpReadListener.handleEventWithNoRunningRequest(HttpReadListener.java:150)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:128)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:56)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.conduits.ReadReadyHandler$ChannelListenerHandler.readReady(ReadReadyHandler.java:66)
	at org.xnio.nio.NioSocketConduit.handleReady(NioSocketConduit.java:89)
	at org.xnio.nio.WorkerThread.run(WorkerThread.java:545)
[2018-01-25 23:21:52,767][DEBUG][io.undertow.request.io] Error reading request
java.io.IOException: Connection reset by peer
	at sun.nio.ch.FileDispatcherImpl.read0(Native Method)
	at sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:39)
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:223)
	at sun.nio.ch.IOUtil.read(IOUtil.java:192)
	at sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:380)
	at org.xnio.nio.NioSocketConduit.read(NioSocketConduit.java:287)
	at org.xnio.conduits.ConduitStreamSourceChannel.read(ConduitStreamSourceChannel.java:127)
	at io.undertow.server.protocol.http.HttpReadListener.handleEventWithNoRunningRequest(HttpReadListener.java:150)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:128)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:56)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.conduits.ReadReadyHandler$ChannelListenerHandler.readReady(ReadReadyHandler.java:66)
	at org.xnio.nio.NioSocketConduit.handleReady(NioSocketConduit.java:89)
	at org.xnio.nio.WorkerThread.run(WorkerThread.java:545)
[2018-01-26 00:11:58,998][DEBUG][io.undertow.request.io] UT005014: Failed to parse HTTP request
java.lang.IllegalArgumentException: Invalid string contents
	at io.undertow.util.HttpString.<init>(HttpString.java:111)
	at io.undertow.util.HttpString.<init>(HttpString.java:101)
	at io.undertow.server.protocol.http.HttpRequestParser$$generated.handleHttpVerb(Unknown Source)
	at io.undertow.server.protocol.http.HttpRequestParser.handle(HttpRequestParser.java:213)
	at io.undertow.server.protocol.http.HttpReadListener.handleEventWithNoRunningRequest(HttpReadListener.java:176)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:128)
	at io.undertow.server.protocol.http.HttpOpenListener.handleEvent(HttpOpenListener.java:143)
	at io.undertow.server.protocol.http.HttpOpenListener.handleEvent(HttpOpenListener.java:90)
	at io.undertow.server.protocol.http.HttpOpenListener.handleEvent(HttpOpenListener.java:49)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.ChannelListeners$10.handleEvent(ChannelListeners.java:291)
	at org.xnio.ChannelListeners$10.handleEvent(ChannelListeners.java:286)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.nio.NioTcpServerHandle.handleReady(NioTcpServerHandle.java:53)
	at org.xnio.nio.WorkerThread.run(WorkerThread.java:545)
[2018-01-26 01:28:34,817][DEBUG][io.undertow.request.io] UT005014: Failed to parse HTTP request
java.lang.IllegalArgumentException: Invalid string contents
	at io.undertow.util.HttpString.<init>(HttpString.java:111)
	at io.undertow.util.HttpString.<init>(HttpString.java:101)
	at io.undertow.server.protocol.http.HttpRequestParser$$generated.handleHttpVerb(Unknown Source)
	at io.undertow.server.protocol.http.HttpRequestParser.handle(HttpRequestParser.java:213)
	at io.undertow.server.protocol.http.HttpReadListener.handleEventWithNoRunningRequest(HttpReadListener.java:176)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:128)
	at io.undertow.server.protocol.http.HttpOpenListener.handleEvent(HttpOpenListener.java:143)
	at io.undertow.server.protocol.http.HttpOpenListener.handleEvent(HttpOpenListener.java:90)
	at io.undertow.server.protocol.http.HttpOpenListener.handleEvent(HttpOpenListener.java:49)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.ChannelListeners$10.handleEvent(ChannelListeners.java:291)
	at org.xnio.ChannelListeners$10.handleEvent(ChannelListeners.java:286)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.nio.NioTcpServerHandle.handleReady(NioTcpServerHandle.java:53)
	at org.xnio.nio.WorkerThread.run(WorkerThread.java:545)
[2018-01-26 01:28:35,360][DEBUG][io.undertow.request.io] UT005014: Failed to parse HTTP request
java.lang.IllegalArgumentException: Invalid string contents
	at io.undertow.util.HttpString.<init>(HttpString.java:111)
	at io.undertow.util.HttpString.<init>(HttpString.java:101)
	at io.undertow.server.protocol.http.HttpRequestParser$$generated.handleHttpVerb(Unknown Source)
	at io.undertow.server.protocol.http.HttpRequestParser.handle(HttpRequestParser.java:213)
	at io.undertow.server.protocol.http.HttpReadListener.handleEventWithNoRunningRequest(HttpReadListener.java:176)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:128)
	at io.undertow.server.protocol.http.HttpOpenListener.handleEvent(HttpOpenListener.java:143)
	at io.undertow.server.protocol.http.HttpOpenListener.handleEvent(HttpOpenListener.java:90)
	at io.undertow.server.protocol.http.HttpOpenListener.handleEvent(HttpOpenListener.java:49)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.ChannelListeners$10.handleEvent(ChannelListeners.java:291)
	at org.xnio.ChannelListeners$10.handleEvent(ChannelListeners.java:286)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.nio.NioTcpServerHandle.handleReady(NioTcpServerHandle.java:53)
	at org.xnio.nio.WorkerThread.run(WorkerThread.java:545)
[2018-01-26 01:28:35,907][DEBUG][io.undertow.request.io] UT005014: Failed to parse HTTP request
java.lang.IllegalArgumentException: Invalid string contents
	at io.undertow.util.HttpString.<init>(HttpString.java:111)
	at io.undertow.util.HttpString.<init>(HttpString.java:101)
	at io.undertow.server.protocol.http.HttpRequestParser$$generated.handleHttpVerb(Unknown Source)
	at io.undertow.server.protocol.http.HttpRequestParser.handle(HttpRequestParser.java:213)
	at io.undertow.server.protocol.http.HttpReadListener.handleEventWithNoRunningRequest(HttpReadListener.java:176)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:128)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:56)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.conduits.ReadReadyHandler$ChannelListenerHandler.readReady(ReadReadyHandler.java:66)
	at org.xnio.nio.NioSocketConduit.handleReady(NioSocketConduit.java:89)
	at org.xnio.nio.WorkerThread.run(WorkerThread.java:545)
[2018-01-26 01:28:36,437][DEBUG][io.undertow.request.io] UT005014: Failed to parse HTTP request
java.lang.IllegalArgumentException: Invalid string contents
	at io.undertow.util.HttpString.<init>(HttpString.java:111)
	at io.undertow.util.HttpString.<init>(HttpString.java:101)
	at io.undertow.server.protocol.http.HttpRequestParser$$generated.handleHttpVerb(Unknown Source)
	at io.undertow.server.protocol.http.HttpRequestParser.handle(HttpRequestParser.java:213)
	at io.undertow.server.protocol.http.HttpReadListener.handleEventWithNoRunningRequest(HttpReadListener.java:176)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:128)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:56)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.conduits.ReadReadyHandler$ChannelListenerHandler.readReady(ReadReadyHandler.java:66)
	at org.xnio.nio.NioSocketConduit.handleReady(NioSocketConduit.java:89)
	at org.xnio.nio.WorkerThread.run(WorkerThread.java:545)
[2018-01-26 01:28:36,976][DEBUG][io.undertow.request.io] UT005014: Failed to parse HTTP request
java.lang.IllegalArgumentException: Invalid string contents
	at io.undertow.util.HttpString.<init>(HttpString.java:111)
	at io.undertow.util.HttpString.<init>(HttpString.java:101)
	at io.undertow.server.protocol.http.HttpRequestParser$$generated.handleHttpVerb(Unknown Source)
	at io.undertow.server.protocol.http.HttpRequestParser.handle(HttpRequestParser.java:213)
	at io.undertow.server.protocol.http.HttpReadListener.handleEventWithNoRunningRequest(HttpReadListener.java:176)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:128)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:56)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.conduits.ReadReadyHandler$ChannelListenerHandler.readReady(ReadReadyHandler.java:66)
	at org.xnio.nio.NioSocketConduit.handleReady(NioSocketConduit.java:89)
	at org.xnio.nio.WorkerThread.run(WorkerThread.java:545)
[2018-01-26 01:28:37,535][DEBUG][io.undertow.request.io] UT005014: Failed to parse HTTP request
java.lang.IllegalArgumentException: Invalid string contents
	at io.undertow.util.HttpString.<init>(HttpString.java:111)
	at io.undertow.util.HttpString.<init>(HttpString.java:101)
	at io.undertow.server.protocol.http.HttpRequestParser$$generated.handleHttpVerb(Unknown Source)
	at io.undertow.server.protocol.http.HttpRequestParser.handle(HttpRequestParser.java:213)
	at io.undertow.server.protocol.http.HttpReadListener.handleEventWithNoRunningRequest(HttpReadListener.java:176)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:128)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:56)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.conduits.ReadReadyHandler$ChannelListenerHandler.readReady(ReadReadyHandler.java:66)
	at org.xnio.nio.NioSocketConduit.handleReady(NioSocketConduit.java:89)
	at org.xnio.nio.WorkerThread.run(WorkerThread.java:545)
[2018-01-26 01:28:38,078][DEBUG][io.undertow.request.io] UT005014: Failed to parse HTTP request
java.lang.IllegalArgumentException: Invalid string contents
	at io.undertow.util.HttpString.<init>(HttpString.java:111)
	at io.undertow.util.HttpString.<init>(HttpString.java:101)
	at io.undertow.server.protocol.http.HttpRequestParser$$generated.handleHttpVerb(Unknown Source)
	at io.undertow.server.protocol.http.HttpRequestParser.handle(HttpRequestParser.java:213)
	at io.undertow.server.protocol.http.HttpReadListener.handleEventWithNoRunningRequest(HttpReadListener.java:176)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:128)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:56)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.conduits.ReadReadyHandler$ChannelListenerHandler.readReady(ReadReadyHandler.java:66)
	at org.xnio.nio.NioSocketConduit.handleReady(NioSocketConduit.java:89)
	at org.xnio.nio.WorkerThread.run(WorkerThread.java:545)
[2018-01-26 01:28:38,644][DEBUG][io.undertow.request.io] UT005014: Failed to parse HTTP request
java.lang.IllegalArgumentException: Invalid string contents
	at io.undertow.util.HttpString.<init>(HttpString.java:111)
	at io.undertow.util.HttpString.<init>(HttpString.java:101)
	at io.undertow.server.protocol.http.HttpRequestParser$$generated.handleHttpVerb(Unknown Source)
	at io.undertow.server.protocol.http.HttpRequestParser.handle(HttpRequestParser.java:213)
	at io.undertow.server.protocol.http.HttpReadListener.handleEventWithNoRunningRequest(HttpReadListener.java:176)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:128)
	at io.undertow.server.protocol.http.HttpOpenListener.handleEvent(HttpOpenListener.java:143)
	at io.undertow.server.protocol.http.HttpOpenListener.handleEvent(HttpOpenListener.java:90)
	at io.undertow.server.protocol.http.HttpOpenListener.handleEvent(HttpOpenListener.java:49)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.ChannelListeners$10.handleEvent(ChannelListeners.java:291)
	at org.xnio.ChannelListeners$10.handleEvent(ChannelListeners.java:286)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.nio.NioTcpServerHandle.handleReady(NioTcpServerHandle.java:53)
	at org.xnio.nio.WorkerThread.run(WorkerThread.java:545)
[2018-01-26 01:28:39,189][DEBUG][io.undertow.request.io] UT005014: Failed to parse HTTP request
java.lang.IllegalArgumentException: Invalid string contents
	at io.undertow.util.HttpString.<init>(HttpString.java:111)
	at io.undertow.util.HttpString.<init>(HttpString.java:101)
	at io.undertow.server.protocol.http.HttpRequestParser$$generated.handleHttpVerb(Unknown Source)
	at io.undertow.server.protocol.http.HttpRequestParser.handle(HttpRequestParser.java:213)
	at io.undertow.server.protocol.http.HttpReadListener.handleEventWithNoRunningRequest(HttpReadListener.java:176)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:128)
	at io.undertow.server.protocol.http.HttpOpenListener.handleEvent(HttpOpenListener.java:143)
	at io.undertow.server.protocol.http.HttpOpenListener.handleEvent(HttpOpenListener.java:90)
	at io.undertow.server.protocol.http.HttpOpenListener.handleEvent(HttpOpenListener.java:49)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.ChannelListeners$10.handleEvent(ChannelListeners.java:291)
	at org.xnio.ChannelListeners$10.handleEvent(ChannelListeners.java:286)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.nio.NioTcpServerHandle.handleReady(NioTcpServerHandle.java:53)
	at org.xnio.nio.WorkerThread.run(WorkerThread.java:545)
[2018-01-26 01:28:39,759][DEBUG][io.undertow.request.io] UT005014: Failed to parse HTTP request
java.lang.IllegalArgumentException: Invalid string contents
	at io.undertow.util.HttpString.<init>(HttpString.java:111)
	at io.undertow.util.HttpString.<init>(HttpString.java:101)
	at io.undertow.server.protocol.http.HttpRequestParser$$generated.handleHttpVerb(Unknown Source)
	at io.undertow.server.protocol.http.HttpRequestParser.handle(HttpRequestParser.java:213)
	at io.undertow.server.protocol.http.HttpReadListener.handleEventWithNoRunningRequest(HttpReadListener.java:176)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:128)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:56)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.conduits.ReadReadyHandler$ChannelListenerHandler.readReady(ReadReadyHandler.java:66)
	at org.xnio.nio.NioSocketConduit.handleReady(NioSocketConduit.java:89)
	at org.xnio.nio.WorkerThread.run(WorkerThread.java:545)
[2018-01-26 01:28:40,309][DEBUG][io.undertow.request.io] UT005014: Failed to parse HTTP request
java.lang.IllegalArgumentException: Invalid string contents
	at io.undertow.util.HttpString.<init>(HttpString.java:111)
	at io.undertow.util.HttpString.<init>(HttpString.java:101)
	at io.undertow.server.protocol.http.HttpRequestParser$$generated.handleHttpVerb(Unknown Source)
	at io.undertow.server.protocol.http.HttpRequestParser.handle(HttpRequestParser.java:213)
	at io.undertow.server.protocol.http.HttpReadListener.handleEventWithNoRunningRequest(HttpReadListener.java:176)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:128)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:56)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.conduits.ReadReadyHandler$ChannelListenerHandler.readReady(ReadReadyHandler.java:66)
	at org.xnio.nio.NioSocketConduit.handleReady(NioSocketConduit.java:89)
	at org.xnio.nio.WorkerThread.run(WorkerThread.java:545)
[2018-01-26 01:28:40,839][DEBUG][io.undertow.request.io] UT005014: Failed to parse HTTP request
java.lang.IllegalArgumentException: Invalid string contents
	at io.undertow.util.HttpString.<init>(HttpString.java:111)
	at io.undertow.util.HttpString.<init>(HttpString.java:101)
	at io.undertow.server.protocol.http.HttpRequestParser$$generated.handleHttpVerb(Unknown Source)
	at io.undertow.server.protocol.http.HttpRequestParser.handle(HttpRequestParser.java:213)
	at io.undertow.server.protocol.http.HttpReadListener.handleEventWithNoRunningRequest(HttpReadListener.java:176)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:128)
	at io.undertow.server.protocol.http.HttpOpenListener.handleEvent(HttpOpenListener.java:143)
	at io.undertow.server.protocol.http.HttpOpenListener.handleEvent(HttpOpenListener.java:90)
	at io.undertow.server.protocol.http.HttpOpenListener.handleEvent(HttpOpenListener.java:49)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.ChannelListeners$10.handleEvent(ChannelListeners.java:291)
	at org.xnio.ChannelListeners$10.handleEvent(ChannelListeners.java:286)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.nio.NioTcpServerHandle.handleReady(NioTcpServerHandle.java:53)
	at org.xnio.nio.WorkerThread.run(WorkerThread.java:545)
[2018-01-26 01:28:41,371][DEBUG][io.undertow.request.io] UT005014: Failed to parse HTTP request
java.lang.IllegalArgumentException: Invalid string contents
	at io.undertow.util.HttpString.<init>(HttpString.java:111)
	at io.undertow.util.HttpString.<init>(HttpString.java:101)
	at io.undertow.server.protocol.http.HttpRequestParser$$generated.handleHttpVerb(Unknown Source)
	at io.undertow.server.protocol.http.HttpRequestParser.handle(HttpRequestParser.java:213)
	at io.undertow.server.protocol.http.HttpReadListener.handleEventWithNoRunningRequest(HttpReadListener.java:176)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:128)
	at io.undertow.server.protocol.http.HttpOpenListener.handleEvent(HttpOpenListener.java:143)
	at io.undertow.server.protocol.http.HttpOpenListener.handleEvent(HttpOpenListener.java:90)
	at io.undertow.server.protocol.http.HttpOpenListener.handleEvent(HttpOpenListener.java:49)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.ChannelListeners$10.handleEvent(ChannelListeners.java:291)
	at org.xnio.ChannelListeners$10.handleEvent(ChannelListeners.java:286)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.nio.NioTcpServerHandle.handleReady(NioTcpServerHandle.java:53)
	at org.xnio.nio.WorkerThread.run(WorkerThread.java:545)
[2018-01-26 01:28:41,929][DEBUG][io.undertow.request.io] UT005014: Failed to parse HTTP request
java.lang.IllegalArgumentException: Invalid string contents
	at io.undertow.util.HttpString.<init>(HttpString.java:111)
	at io.undertow.util.HttpString.<init>(HttpString.java:101)
	at io.undertow.server.protocol.http.HttpRequestParser$$generated.handleHttpVerb(Unknown Source)
	at io.undertow.server.protocol.http.HttpRequestParser.handle(HttpRequestParser.java:213)
	at io.undertow.server.protocol.http.HttpReadListener.handleEventWithNoRunningRequest(HttpReadListener.java:176)
	at io.undertow.server.protocol.http.HttpReadListener.handleEvent(HttpReadListener.java:128)
	at io.undertow.server.protocol.http.HttpOpenListener.handleEvent(HttpOpenListener.java:143)
	at io.undertow.server.protocol.http.HttpOpenListener.handleEvent(HttpOpenListener.java:90)
	at io.undertow.server.protocol.http.HttpOpenListener.handleEvent(HttpOpenListener.java:49)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.ChannelListeners$10.handleEvent(ChannelListeners.java:291)
	at org.xnio.ChannelListeners$10.handleEvent(ChannelListeners.java:286)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.nio.NioTcpServerHandle.handleReady(NioTcpServerHandle.java:53)
	at org.xnio.nio.WorkerThread.run(WorkerThread.java:545)
